src/remarx/sandbox_utils.py

```diff
@@ -35,7 +36,7 @@ def create_client() -> AzureOpenAI:
 
 
 def submit_prompt(
-    task_prompt, user_prompt, model="gpt-4o", client=None
+    task_prompt, user_prompt, model="gpt-4o", client: Optional[AzureOpenAI] = None
```

Comment:
Happy to use `Optional` but also open to this instead

```suggestion
    task_prompt, user_prompt, model="gpt-4o", client: None | AzureOpenAI = None
```